ycms/cms/templates/ward/ward_management.html

```diff
@@ -0,0 +1,87 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% block title %}
+    {% translate "Ward Management" %}
+{% endblock title %}
+{% block content %}
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "YCMS at a glance" %}</div>
+            <div class="grid grid-cols-3">
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Wards" %}</div>
+                    <div class="text-3xl font-bold">{{ wards.all_wards_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Doctors" %}</div>
+                    <div class="text-3xl font-bold">{{ wards.all_doctors_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Nurses" %}</div>
+                    <div class="text-3xl font-bold">{{ wards.all_nurses_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ wards.all_beds_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Available Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ wards.all_available_beds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Occupied Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ wards.all_occupied_beds }}</div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br>
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "Add a New Ward" %}</div>
+            <br>
+            <form enctype="multipart/form-data"
+                  method="post"
+                  id="ward_form"
+                  class="mb-12 flex justify-between items-center"
+                  data-unsaved-warning>
+                {% csrf_token %}
+                {% for field in ward_form %}
+                    <div>
+                        <label for="{{ field.id_for_label }}"
```

Comment:
```suggestion
                  class="mb-12 flex justify-between items-center gap-4"
                  data-unsaved-warning>
                {% csrf_token %}
                {% for field in ward_form %}
                    <div class="grow">
                        <label for="{{ field.id_for_label }}"
```
Sorry, I missed this when looking at it on a smaller screen